taipan-ms/src/main/resources/application.yml

```diff
@@ -0,0 +1,29 @@
+### Spring properties
+# Service registers under this name
+spring:
+  application:
+    name: taipan-ms
+
+# Discovery Server Access
+eureka:
+  instance:
+    metadataMap:
+      type: executer
+      friendlyname: Executer
+  client:
+    serviceUrl:
+      defaultZone: http://localhost:1111/eureka/
```

Comment:
The other pull request requires url of Eureka server in different format with different port number.

Reply:
jep, but for this master state is everything ok